### PR TITLE
Add triage-2021-02-24

### PR DESCRIPTION
**Pull request by rylev**

### CODE REVIEW

File: triage/2021-02-24.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+#### Improvements
+
+Only store a LocalDefId in some HIR nodes [#81611](https://github.com/rust-lang/rust/issues/81611)
+- Very large improvement in [instruction counts](https://perf.rust-lang.org/compare.html?start=a143517d44cac50b20cbd3a0b579addab40dd399&end=8fe989dd768f5dfdb0fc90933f3f74fa4579fefd&stat=instructions:u) (up to -9.6% on `incr-unchanged` builds of `many-assoc-items-check`)
+- Large improvements in the associated_item query which naturally led to improvements in the stress test related to associated items.
+
+Inline try_get_cached [#82197](https://github.com/rust-lang/rust/issues/82197)
+- Very large improvement in [instruction counts](https://perf.rust-lang.org/compare.html?start=8fe989dd768f5dfdb0fc90933f3f74fa4579fefd&end=ee88f46bb5e27c4d9f30326e69ff2298dcbeecb1&stat=instructions:u) (up to -10.4% on `full` builds of `externs-debug`)
+- An attempt to fix an issue due to a bad interaction with [#81892](https://github.com/rust-lang/rust/issues/81892) which removed a bunch of largely unnecessary inline directives.
```
Comment (cjgillot) [inline triage/2021-02-24.md:26]: Should the formulation attribute the improvement to the three PRs ? https://github.com/rust-lang/rust/issues/82197 , https://github.com/rust-lang/rust/issues/81892 and https://github.com/rust-lang/rust/issues/81855

Comment (rylev) [reply triage/2021-02-24.md:26]: The improvement itself is just from what was measured after the merging of rust-lang/rust#82197, but I can reference https://github.com/rust-lang/rust/pull/81855 if you'd like.

Comment (tmiasko) [reply triage/2021-02-24.md:26]: The changes from rust-lang/rust#81855 are directly responsible for the improvement. I would appreciate if you could emphasize that. I would feel uncomfortable if the improvements were to be attributed to rust-lang/rust#82197 which merely added an inline hint to fix unintended interaction with rust-lang/rust#81892, which landed in the meantime. Thanks.

Comment (rylev) [reply triage/2021-02-24.md]: @cjgillot @tmiasko I've adjusted the wording. Thoughts?

Comment (tmiasko) [reply triage/2021-02-24.md]: I would describe it as:

> rust-lang/rust#82197 shows improvements originally expected from rust-lang/rust#81855, which based query fast path on `try_get_cached`. In-between the initial successful perf run of rust-lang/rust#81855 and merging, another PR rust-lang/rust#81892 removed an inline hint from the function, which was soon to be on the fast path.

**Pull request merged**